hack/test-keystone.sh

```diff
@@ -313,31 +314,350 @@ runTests() {
   # kube::test::get_object_assert "tenants test" "{{.status.phase}}" 'Terminating'
   kube::test::get_object_assert "tenants" "{{range.items}}{{$id_field}}:{{end}}" ''
 
+  kubectl create -f docs/user-guide/te.yaml "${kube_admin_flags[@]}"
+  kube::test::get_object_assert "tenants" "{{range.items}}{{$id_field}}:{{end}}" 'test:'
+  kubectl create -f docs/user-guide/ns.yaml "${kube_user_flags[@]}"
+  kube::test::get_object_assert "namespaces" "{{range.items}}{{$id_field}}:{{end}}" 'test:'
+
+  kubectl delete tenant test "${kube_admin_flags[@]}" --grace-period=0
+  sleep 5
+  kube::test::get_object_assert "tenants" "{{range.items}}{{$id_field}}:{{end}}" ''
+  kube::test::get_object_assert "namespaces" "{{range.items}}{{$id_field}}:{{end}}" ''
+
   #################################
   # namespace creation / deletion #
   #################################
+  kube::log::status "Testing kubectl(${version}:namespaces)"
+
   kube_flags=${kube_admin_flags[@]}
   kube::test::get_object_assert "namespaces" "{{range.items}}{{$id_field}}:{{end}}" 'default:'
   kube_flags=${kube_user_flags[@]}
   kube::test::get_object_assert "namespaces" "{{range.items}}{{$id_field}}:{{end}}" ''
 
   # Create 'test' tenant
-  # kubectl create -f docs/user-guide/te.yaml "${kube_admin_flags[@]}"
+  kubectl create -f docs/user-guide/te.yaml "${kube_admin_flags[@]}"
   # Create 'test' namespace for 'test' tenant
-  # kubectl create -f docs/user-guide/ns.yaml "${kube_user_flags[@]}"
-  # kube::test::get_object_assert "namespaces" "{{range.items}}{{$id_field}}:{{end}}" 'test:'
+  kubectl create -f docs/user-guide/ns.yaml "${kube_user_flags[@]}"
```

Comment:
1. There should be tests to assert `tenant` of created resources
2. `kube::test::get....` should also query resources in dedicated tenant